webui/react/src/components/UPlot/UPlotScatter.tsx

```diff
@@ -1,11 +1,11 @@
 import React, { useMemo, useRef } from 'react';
 import uPlot from 'uplot';
 
-import { Scale } from 'components/ScaleSelectFilter';
 import UPlotChart, { Options } from 'components/UPlot/UPlotChart';
 import QuadTree, { pointWithin } from 'components/UPlot/UPlotScatter/quadtree';
 
 import { Range } from '../../shared/types';
+import { Scale } from '../../types';
```

Comment:
**suggestion:** just a FYI, the two imports above can reference from an "absolute" path given how our webpack / typescript has aliases setup. So they can the following instead:
```
import { Range } from 'shared/types';
import { Scale } from 'types';
```